packages/as-transformer/src/transformer.ts

```diff
@@ -206,33 +164,43 @@ export class Transformer extends TransformVisitor {
    * @param parser - A {@link Parser} object.
    */
   afterParse(parser: Parser): void {
+    Debug.log('Starting after parse.');
     let sources = parser.sources.filter(
       // Fetching only project parsed sources (AST Tree for each file)
-      (source) =>
-        !source.internalPath.startsWith(`node_modules/`) &&
-        !utils.isLibrary(source) &&
-        GlobalUpdates.get().filter(
-          (update) =>
-            (update.from === 'MassaExport' &&
-              source.internalPath.includes(
-                update.data.get('funcToPrivate')![0] + 'Wrapper',
-              )) ||
-            (update.from === 'as-transformer' &&
-              source.internalPath.includes(update.content)),
-        ).length <= 0 &&
-        !source.internalPath.includes('build/'),
+      (source) => {
+        return (
+          !source.internalPath.startsWith(`node_modules/`) &&
+          !utils.isLibrary(source) &&
+          /* GlobalUpdates.get().filter(
+            (update) =>
+              (update.from === 'MassaExport' &&
+                (source.internalPath.includes(
+                  update.data.get('funcToPrivate')![0] + 'Helper',
+                ) ||
+                  source.internalPath.includes(
+                    update.data.get('funcToPrivate')![0] + 'RHelper',
+                  ))) ||
+              (update.from === 'as-transformer' &&
+                source.internalPath.includes(update.content)),
+          ).length <= 0 && */
```

Comment:
Why is this commented? If you want to keep it, please add a comment explaining why.

Reply:
same here